mantidproject/mantid
# Fix chopper gap unit in ILL reflectometry.

**Description of work.**

This overrides the sample logs of chopper gap and position in a correct unit regardless of the unit in nexus file, which can be cm or m depending on the acquisition cycle. This was later on messing up with the resolution calculation.

**To test:**

Load the files 541838.nxs and 397812.nxs from the unit test data under ILL/D17.
Make sure they both have a sample log named `Distance.ChopperGap` with a value of 0.082 and units in meter.



*There is no associated issue.*



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
